hashicorp-vault/variables.tf

```diff
@@ -0,0 +1,89 @@
+variable "cluster_name" {
+  type        = string
+  description = "A name of the EKS cluster"
+}
+
+variable "chart_version" {
+  type        = string
+  description = "A Helm Chart version"
+  default     = "0.17.0"
+}
+
+variable "chart_name" {
+  type        = string
+  default     = "hashicorp-vault"
+  description = "A chart name for vault installation"
+}
+
+variable "chart_namespace" {
+  type        = string
+  default     = "default"
```

Comment:
default namespace allow access to anybody who has access to the cluster, much better used kube-system namespace